src/node_options.cc

```diff
@@ -667,9 +667,10 @@ PerProcessOptionsParser::PerProcessOptionsParser(
             kAllowedInEnvironment);
   AddOption("--report-directory",
             "define custom report pathname."
-            " (default: current working directory of Node.js process)",
+            " (default: current working directory)",
             &PerProcessOptions::report_directory,
             kAllowedInEnvironment);
+  AddAlias("--report-dir", "--report-directory");
```

Comment:
I would switch, so that the "legacy" (long) name is aliased to the soon-to-be-preferred, consistent name.